maskrcnn_benchmark/utils/metric_logger.py

```diff
@@ -61,3 +65,42 @@ def __str__(self):
                 "{}: {:.4f} ({:.4f})".format(name, meter.median, meter.global_avg)
             )
         return self.delimiter.join(loss_str)
+
+
+class TensorboardLogger(MetricLogger):
+    def __init__(self,
+                 log_dir='logs',
+                 exp_name='maskrcnn-benchmark',
```

Comment:
I still think that there is no need for two arguments: if you run two trainings on the same directory, it will have conflicts with the checkpoints and it will not work as expected.

In this case, I would just pass the full path to the file as an argument.